aws-auth/src/main/java/software/amazon/awssdk/awsauth/credentials/profile/Profile.java

```diff
@@ -13,7 +13,7 @@
  * permissions and limitations under the License.
  */
 
-package software.amazon.awssdk.auth.profile;
+package software.amazon.awssdk.awsauth.credentials.profile;
```

Comment:
The profile stuff may be better place in it's own module. It isn't just credentials and region and we may expand on it in the future to include other profile configuration.